soca/obs/sss_smap_jpl.yaml

```diff
@@ -22,13 +22,17 @@
     maxvalue: 40.0
   - filter: Background Check
     threshold: 5.0
-  - filter: Domain Check
-    action:
-      name: passivate
-    where:
-    - variable: {name: sea_surface_temperature@GeoVaLs}
-      minvalue: 10.0
+#  - filter: Domain Check
```

Comment:
sss should be passive for now, re-activate this filter.

Reply:
@guillaumevernieres, this yaml is being used for the 1 deg experiment with these obs assimilated. There is another run where this filter is activated.

Reply:
@jakir, I'm asking you to remove the commented out filter. The default configuration in develop should not assimilate sss, until we understand errors and biases from these obs.